io.github.fheroes2.yaml

```diff
@@ -0,0 +1,31 @@
+app-id: io.github.fheroes2
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+finish-args:
+  - --filesystem=home
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+command: fheroes2
+modules:
+  - name: fheroes2
+    buildsystem: cmake
+    config-opts:
+        - "-DSDL2MAIN_LIBRARY="
+    sources:
+      - type: git
+        url: https://github.com/ihhub/fheroes2
+        branch: master
+        commit: 4ce7b1148b0c2ec4e5da67aeb79522942c7d6a95
+      - type: file
+        path: io.github.fheroes2.metainfo.xml
+      - type: file
+        path: AppIcon128.png
```

Comment:
there is an icon the git repository, so you shouldn't need to add it here. while it's not installed (it's a bug) you can directly install it

Reply:
https://github.com/ihhub/fheroes2/pull/5405

This one should remove the bug.

Reply:
but you don't install it.